.mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
```

Comment:
Nit: Since there's no typing on this repo and the mypy target is turned off in the Makefile, this file isn't actually needed (but isn't doing any harm either)

Reply:
Good point. I've removed it and renamed the commit so that it doesn't suggest mypy has been enabled